utils/check-missing-logos.js

```diff
@@ -0,0 +1,46 @@
+const {
```

Comment:
As much as I love a utility script, I think this is only good for a one time use since the repository automation will handle keeping the rest of them schema compliant. If you think this is still worth adding to the repo, let's get a package.json script setup to execute it for ease of use.

Reply:
@aswanson-nr Ye the reason i did this was so that the content team could run it to check they had added all the missing logos, they can pull down the PR and work on it from there. Probably also why the checks are failing, they will fail until these are added (except the unit tests i'll fix those). Would it make sense to either 

A: Use a Feature Branch for this or
B: Leave this in draft so the content team can do work using the script

Reply:
We went with A, it's done